data-structures-stacks/get-2nd-from-top.js

```diff
@@ -0,0 +1,20 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  const placeholder = [];
+  let item;
+  for (let i = 0; i < 2; i++) {
+    item = stack.pop();
+    if (item === undefined && i === 0) {
+      return undefined;
+    }
+    if (item === undefined) {
+      stack.push(placeholder[0]);
+      return item;
+    }
+    placeholder.push(item);
+  }
```

Comment:
This can be done without a loop. If you have the time, try refactoring to work without a loop